quarkus-picocli/src/test/java/PicocliDevIT.java

```diff
@@ -6,41 +6,49 @@
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.services.QuarkusApplication;
 import io.quarkus.ts.qe.command.AgeCommand;
+import io.quarkus.ts.qe.command.CommonOptions;
 import io.quarkus.ts.qe.command.EntryCommand;
 import io.quarkus.ts.qe.command.HelloCommand;
-import io.quarkus.ts.qe.configuration.Config;
+import io.quarkus.ts.qe.command.OtherCommand;
+import io.quarkus.ts.qe.command.OtherEntryCommand;
+import io.quarkus.ts.qe.configuration.Configuration;
 import io.quarkus.ts.qe.services.AgeService;
 import io.quarkus.ts.qe.services.HelloService;
 
 @QuarkusScenario
 public class PicocliDevIT {
```

Comment:
Reason why this test is not excluded in native mode is that you don't follow following pattern: https://github.com/quarkus-qe/quarkus-test-suite/blob/9ea57d027cda27e5815faa58bbc35f24a2be0a8b/pom.xml#L705

It doesn't make sense to run a DEV mode test in native. There is nothing native a DEV Mode as they are 2 different launch modes.
```suggestion
public class PicocliDevModeIT {
```

On the other hand, I think you don't realize that `quarkus.profile=dev` is not the same thing as a DEV mode test. Can you maybe clarify what this test does and why is it `Dev` while other is `Prod`? So to make it absolutely clear, by setting this property you are not testing a DEV mode. You are still testing JVM mode or native mode.

Reply:
Hi @michalvavrik , I know that, and sorry about the confusion between Quarkus profiles and launch modes.

I  wanted to test different configurations in quarkus-picocli as described in the guide, such as selecting different Picocli commands based on the active profile. 
I'll rename dev to "test" profile, so both profiles can be executed in native mode.

Reply:
Sounds good, thanks.